file-formats/smbc/examples/z_aff_example_smbc.smbc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "AFF Example Object for SMBC",
+    "originalLanguage": "en"
+  },
+  "serviceConfiguration": {
+    "serviceBinding": "/DMO/SB_CUSTOMER_MSSK_O4",
+    "serviceName": "/DMO/SD_CUSTOMER_MSSK",
+    "serviceVersion": "0001",
+    "rootEntity": "Customer",
+    "name": "Maintain customer",
+    "skipRootEntity": false
+  },
+  "appConfiguration": {
+    "listReport": {},
```

Comment:
Do we want to serialize these empty arrays?

Reply:
If possible, I would skip them by removing `$showAlways`, but the object type owners have to decide how import it is for them to list an empty `appConfiguration`

Reply:
$showAlways was removed